components/events/EventCard.vue

```diff
@@ -65,96 +65,108 @@ export default class extends Vue {
 <style lang="scss" scoped>
 @import '~carbon-components/scss/globals/scss/typography';
 
-.card-link {
-  text-decoration: none;
-}
+$img-medium-width: 13rem;
 
 .event-card {
-  height: 15.88rem;
+  min-height: 13rem;
   width: 100%;
-  margin-bottom: 1rem;
-  background-color: $ui-01;
-  color: $text-01;
-  border-top: 1px solid $ui-03;
+  margin-bottom: $layout-03;
+  background-color: $cool-gray-10;
+  color: $cool-gray-80;
   display: flex;
 
   @include mq($until: medium) {
     height: auto;
-    flex-direction: column-reverse;
-  }
-
-  &__content {
-    flex: 1;
-    margin: 1rem;
-    display: flex;
     flex-direction: column;
-    justify-content: space-between;
   }
 
-  &__title {
-    @include type-style('productive-heading-03');
-    margin-top: 0.31rem;
-  }
-
-  &__subtitle {
-    @include type-style('productive-heading-02');
-  }
-
-  &__location, &__date {
-    @include type-style('body-short-01');
-  }
+  &__image {
+    flex: 0 0 14rem;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
```

Comment:
I'm wondering if we leave background-position _not_ center... and I only ask because it seems like for the Event images, sometimes they have the text top-left aligned...

![Screen Shot 2020-09-25 at 12 01 56 PM](https://user-images.githubusercontent.com/6276074/94300966-002d5600-ff27-11ea-8cde-88e1ec14b4e0.png)


If we removed this center value, the images render better it seems
![Screen Shot 2020-09-25 at 12 01 19 PM](https://user-images.githubusercontent.com/6276074/94301049-2f43c780-ff27-11ea-83e9-380793b8efcd.png)



What do you think? Or are these images that come with these events too unpredictable?

Reply:
Good catch! but I think we have to include @JRussellHuffman on the conversation :). Look, I did what you propose with the images I have:

<img width="1050" alt="Screen Shot 2020-09-28 at 18 51 51" src="https://user-images.githubusercontent.com/17231966/94463893-f40ef600-01bd-11eb-9b75-64a0175f5b1e.png">

We cut the right part of the img...  

The "problem" I see is that now we have an img almost square compared with the previous one:

- Previous one
<img width="778" alt="Screen Shot 2020-09-28 at 19 12 50" src="https://user-images.githubusercontent.com/17231966/94464316-9d55ec00-01be-11eb-92c4-14558217eee0.png">

- New one
<img width="745" alt="Screen Shot 2020-09-28 at 19 13 34" src="https://user-images.githubusercontent.com/17231966/94464399-b65e9d00-01be-11eb-984d-879fbea12c3e.png">

So.. to be able to see all the img I propose 3 options:
- 1 => make the img wider
<img width="750" alt="Screen Shot 2020-09-28 at 19 14 57" src="https://user-images.githubusercontent.com/17231966/94464560-e7d76880-01be-11eb-949f-4f5707c69df7.png">

-2 => center the img to see 100% of the width
<img width="753" alt="Screen Shot 2020-09-28 at 19 16 13" src="https://user-images.githubusercontent.com/17231966/94464690-15241680-01bf-11eb-82cd-a24c05a5d3db.png">

-3 => show the img to cover 100% width and 100% height (I don't like this one)
<img width="749" alt="Screen Shot 2020-09-28 at 19 17 02" src="https://user-images.githubusercontent.com/17231966/94464783-32f17b80-01bf-11eb-9084-3d8e6ef21022.png">

@JRussellHuffman , @techtolentino if you have any other option, let me know. @JRussellHuffman what should I chose?

Reply:
When someone uploads an event picture, are they given particular constraints, or can it be any size? Option 1 would make sense if the image is always the same size. Option 2 makes the most sense for flexibility, but now it looks like there is a margin on the image on 3 sides, which is less than ideal. could we do a background color behind the image in these scenarios? I'll mock something up.

Agreed, option 3 isn't the way we should go.

Reply:
Could we add border above and below on option 2? 
![borders](https://user-images.githubusercontent.com/7633881/94466477-19ccd880-0187-11eb-9479-112bbbcb5db4.jpg)

Reply:
Its still less than idea though. In the future, would it be possible to do any of the following when they first upload the image?

- give a size recommendation
- force a particular size when they are submitting an event
- give them a preview so they can see if it looks cropped
- allow them to crop the image to fit our ratio
 
Or does all that happen on the airtable side?

Reply:
All that happen on the airtable side... maybe we can talk with the event team to find an img size that fits for all of us

Reply:
I agree, this is a result of having varying dimensions when images are uploaded. From a branding and asset standpoint, it is best to _try_ and ensure we can have the same asset treatment, all around, with these images.

I understand that the Airtable is owned by another team, still I agree that if we can help other teams know which are the best dimensions, maybe that can become an adopted practice?

Reply:
Until we talk with the events team, I will implement the second option with the background-color as @JRussellHuffman suggest to be able to merge and not block, are you both agree?

Reply:
sounds good!